tensorflow/core/distributed_runtime/master.cc

```diff
@@ -116,6 +116,21 @@ void Master::GC() {
   }
 }
 
+MasterSession* Master::FindMasterSession(const string& handle) {
+  MasterSession* session = nullptr;
+  {
+    mutex_lock l(mu_);
+    session = gtl::FindPtrOrNull(sessions_, handle);
+    if (session != nullptr) {
+      session->Ref();
+    }
+  }
+  if (session == nullptr) {
+    done(errors::Aborted("Session ", handle, " is not found."));
```

Comment:
`done` isn't defined in the body of this function, so this won't compile. (Also, it will continue to run the request handler after calling the callback, whereas the request handler should terminate immediately after calling `done` and doing any necessary cleanup.)